tycho-its/projects/justj/pom.xml

```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project>
+	<modelVersion>4.0.0</modelVersion>
+	<groupId>org.eclipse.tycho.tycho-its</groupId>
+	<artifactId>parent-justj</artifactId>
+	<version>0.0.1-SNAPSHOT</version>
+	<packaging>pom</packaging>
+	<properties>
+ 		<tycho-version>2.4.0-SNAPSHOT</tycho-version>
+ 		<target-platform>https://download.eclipse.org/releases/2021-03/</target-platform>
+	</properties>
+	<modules>
+		<module>bundle</module>
+		<module>bundle2</module>
+		<module>feature</module>
+		<module>bundle-product</module>
+		<module>feature-product</module>
+	</modules>
+	<repositories>
+		<repository>
+			<id>platform</id>
+			<url>${target-platform}</url>
+			<layout>p2</layout>
+		</repository>
+		<repository>
+			<id>justj</id>
+			<url>https://download.eclipse.org/justj/jres/11/updates/release/11.0.2</url>
+			<layout>p2</layout>
+		</repository>
+	</repositories>
+	<build>
+		<plugins>
+			<plugin>
+				<groupId>org.eclipse.tycho</groupId>
+				<artifactId>tycho-maven-plugin</artifactId>
+				<version>${tycho-version}</version>
+				<extensions>true</extensions>
+			</plugin>
+				<plugin>
+				<groupId>org.eclipse.tycho</groupId>
+				<artifactId>target-platform-configuration</artifactId>
+				<version>${tycho-version}</version>
+				<configuration>
+					<executionEnvironment>none</executionEnvironment>
```

Comment:
This shouldn't be set for bundles; but only for products where we want to control the EE and don't want a.jre.javase units to participate.

Reply:
Well but that the point here, I don't want (or even can as I using pomless builds) set special things for product. If products need any special setup they should use that automatically and not fail with strange errors... and here we are at the beginning of the discussion, if I don not set anything it don't work either...

And even if I would set this option for products, this will result in my target resolved twice: Once with and once without that option what is also undesirable as I wan't one target resolution for the whole build...